diff --git a/node_modules/ag-grid-enterprise/dist/ag-grid-enterprise.js b/node_modules/ag-grid-enterprise/dist/ag-grid-enterprise.js
index fae99da..55f76a7 100644
--- a/node_modules/ag-grid-enterprise/dist/ag-grid-enterprise.js
+++ b/node_modules/ag-grid-enterprise/dist/ag-grid-enterprise.js
@@ -81285,26 +81285,7 @@ var _LicenseManager = class _LicenseManager {
     this.md5.init();
   }
   validateLicense() {
-    const licenseDetails = this.getLicenseDetails(_LicenseManager.licenseKey);
-    const currentLicenseName = `AG Grid ${licenseDetails.currentLicenseType === "BOTH" ? "and AG Charts " : ""}Enterprise`;
-    const suppliedLicenseName = licenseDetails.suppliedLicenseType === void 0 ? "" : `AG ${licenseDetails.suppliedLicenseType === "BOTH" ? "Grid and AG Charts" : licenseDetails.suppliedLicenseType === "GRID" ? "Grid" : "Charts"} Enterprise`;
-    if (licenseDetails.missing) {
-      if (!this.isWebsiteUrl() || this.isForceWatermark()) {
-        this.outputMissingLicenseKey(currentLicenseName);
-      }
-    } else if (licenseDetails.expired) {
-      const gridReleaseDate = _LicenseManager.getGridReleaseDate();
-      const formattedReleaseDate = _LicenseManager.formatDate(gridReleaseDate);
-      this.outputExpiredKey(licenseDetails.expiry, formattedReleaseDate, currentLicenseName, suppliedLicenseName);
-    } else if (!licenseDetails.valid) {
-      this.outputInvalidLicenseKey(
-        !!licenseDetails.incorrectLicenseType,
-        currentLicenseName,
-        suppliedLicenseName
-      );
-    } else if (licenseDetails.isTrial && licenseDetails.trialExpired) {
-      this.outputExpiredTrialKey(licenseDetails.expiry, currentLicenseName, suppliedLicenseName);
-    }
+    return;  
   }
   static extractExpiry(license) {
     const restrictionHashed = license.substring(license.lastIndexOf("_") + 1, license.length);
